app/models/manageiq/providers/autosde/storage_manager/autosde_client.rb

```diff
@@ -96,4 +96,15 @@ def configure_openapi_client
       # when set to nil, url refers to config.host, as it should be
     end
   end
+
+  private_class_method def self.enqueue_refresh(target_class, target_id, ems_id, task_id)
+    ManageIQ::Providers::Autosde::StorageManager::EmsRefreshWorkflow.create_job(
+      :target_class   => target_class,
+      :target_id      => target_id,
+      :ems_id         => ems_id,
+      :native_task_id => task_id
+    ).tap do |job|
+      job.signal(:start)
+    end
+  end
```

Comment:
I don't think it makes sense to have this in the `AutosdeClient` class since it isn't related to calling the API.  If you just want a "nicer" looking method you could always define a method on the `ManageIQ::Providers::Autosde::StorageManager::EmsRefreshWorkflow` class that takes positional args and calls start automatically.